CentralComputing/UnitTests/StateMachine_Test.cpp

```diff
@@ -1,53 +1,68 @@
-#include "Sensor.h"
-#include <iostream>
-#include <unistd.h>
-#include "Pod.h"
-#include "StateMachineCompact/stdafx.h"
-#include <string>
-#include <fstream>
-#include <sstream>
-#include <algorithm>
-#include <vector>
-#include <stdlib.h>
+#include "StateMachine_Test.h"
 
-#include <time.h>
+//Include any other dependencies 
 
-using namespace std;
-
-// declaring function, implented later
-void print_help();
-void print_help_instr();
+StateMachine_Test::StateMachine_Test(){
```

Comment:
Alright.  I'll simplify down the state machine testing interface and give a short list of tests it can run from main, and then move Curtis's GUI stuff to a new Pod_Test block that will essentially make an all up test.  I think that would fit both what you want to do with Unit_Tests and what Curtis and I want to do with the whole testing suite